src/styles/menu.css

```diff
@@ -24,7 +24,7 @@
 }
 
 .ui.tabular.menu .active.item {
-  border-color: var(--primary-pms-362_900);
+  border-color: var(--primary-pms-362-900);
```

Comment:
Is it updated in variables? I think before we had `_`. If it got updated in variables then we need to change it for all other components as well (label, header, etc.)

Reply:
Thanks!